.github/workflows/integration_test.yaml

```diff
@@ -5,7 +5,11 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@feat-rockcraft-from-src
     secrets: inherit
     with:
-      modules: '["test_charm", "test_proxy", "test_cos", "test_database", "test_db_migration", "test_default"]'
+      modules: '["test_charm", "test_proxy", "test_cos", "test_database", "test_db_migration", "test_default", "test_django"]'
```

Comment:
is `"test_django"` required here? Perhaps it was missed in the removal?

Reply:
removed, thanks